packs/ibmmq/config.yml

```diff
@@ -0,0 +1,36 @@
+id: 924fd4b3-a6d1-4a6e-9e2c-b598f197f713
+
+# Name of the pack (required)
+name: ibmmq
+title: New Relic IBM MQ Monitor
+description: >-
```

Comment:
@hsinghkalsi we just found an issue with string parsing
can you please replace all `>-` with `|`?

Reply:
Done!